src/client/sandbox/node/live-node-list/wrapper-base.js

```diff
@@ -0,0 +1,36 @@
+const DEFINED_PROPERTIES_COUNT = 10000;
+
+export class LiveNodeListWrapperBase {
+    constructor () {
+        const defineProperty = function (index, isEnumerable) {
+            Object.defineProperty(this, index, {
+                enumerable:   isEnumerable,
+                configurable: true,
+
+                get: function () {
+                    this._refreshNodeList();
+
+                    return this._filteredNodeList[index];
+                }
+            });
+        };
+
+        for (let i = 0; i < DEFINED_PROPERTIES_COUNT; i++)
+            defineProperty.call(this, i, false);
+
+        Object.defineProperty(this, '_defineProperty', { value: defineProperty });
+        Object.defineProperty(this, '_refreshNodeList', {
+            value: () => {
+                throw new Error('Not implemented');
+            }
+        });
+    }
+}
+
+LiveNodeListWrapperBase.prototype = NodeList.prototype;
+
+export const NODE_LIST_WRAPPER = new LiveNodeListWrapperBase();
```

Comment:
[here](https://google.github.io/styleguide/jsguide.html#naming-constant-names) is nice explanation of constants I think. 

> Before choosing constant case, consider whether the field really feels like a deeply immutable constant. For example, if any of that instance's observable state can change, it is almost certainly not a constant. Merely intending to never mutate the object is generally not enough.

I think an instance of the wrapper base class doesn't match this conditions